metron-interface/metron-alerts/src/app/model/table-metadata.ts

```diff
@@ -15,23 +15,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {PageSize, RefreshInterval} from '../alerts/configure-rows/configure-rows-enums';
-import {ColumnMetadata} from './column-metadata';
+import { PageSize } from '../alerts/configure-rows/configure-rows-enums';
+import { ColumnMetadata } from './column-metadata';
 
 export class TableMetadata {
   size = PageSize.TWENTY_FIVE;
-  refreshInterval = RefreshInterval.TEN_MIN;
-  hideResolvedAlerts = true;
-  hideDismissedAlerts = true;
```

Comment:
hideResolvedAlerts = true;
hideDismissedAlerts = true;
were dead code.
refreshInterval was moved to auto-polling.service